### PR TITLE
Keep progressing

### PR DESCRIPTION
**Pull request by Wonay**

### CODE REVIEW

File: scripts/core/agent/agent.js [hunk 1]
```diff
@@ -174,12 +202,17 @@ define([
             decideGoal();
             moveTo(newLocation);
             spendHunger(this.hungerRate);
+            spendEnergy(this.exhaustionRate);
+
+            this.age += 0.1;
```
Comment (jacklehamster) [inline scripts/core/agent/agent.js]: Do we need a "age" variable here that keeps increasing? I normally just have a "born" time and the age is simply now - born.

File: scripts/core/agent/agent.js [hunk 2]
```diff
@@ -127,6 +129,26 @@ define([
...
+            // curiosity
+            myself.curiosity = myself.generator.getInt(10, 90);
+            myself.looseCuriosityWithAgeCoef = myself.generator.getFloatInRange(0.01, 0.99);
+
+            // playful
+            myself.playful = myself.generator.getInt(10, 90);
+            myself.loosePlayfulWithAgeCoef = myself.generator.getFloatInRange(0.01, 0.99);
+        }
+
+        var looseFun = function() {
```
Comment (jacklehamster) [inline scripts/core/agent/agent.js]: What does looseFun mean?

Comment (Wonay) [inline scripts/core/agent/agent.js]: It's when age increase curiosity and playfulness decrease.

File: scripts/core/agent/agent.js (hunk 1 above)
Comment (Wonay) [inline scripts/core/agent/agent.js]: Yeah but there is no "time" here. It's per cycle.

**Pull request merged**